src/layout/Navbar/Navbar.scss

```diff
@@ -0,0 +1,122 @@
+@import '../../scss/variables.scss';
+@import '../../scss/colors.scss';
+@import '../../scss/borders.scss';
+
+.navbar {
+  display: flex;
+  flex: 1 0 $navbarWidth;
+  height: 100%;
+  margin: 0;
+  position: absolute;
+  border-right: $tertiaryBorder;
+  transform: translateX(-100%);
+  transition: transform 0.5s ease-in-out;
+  top: 0;
+  left: 0;
+  width: $navbarWidth;
+  z-index: 3;
```

Comment:
navbar should be placed in the `overlay_container`, otherwise we will have bugs when elements will overlap the `navbar`

Reply:
If I'll move it to overlay_container then I wouldn't be able to adjust the main accordingly.
* the right panel, in "create job" should be moved to the  overlay_container